content/docs/guide/gamemodes/climb.md

```diff
@@ -8,19 +8,42 @@ tags:
+Climb started in Counter-strike 1.1. A mapper named Kreedz decided to make a fun mess-around map that did not have the usual defuse objective for his friends. Kreedz enjoys climbing as a hobby, so he felt that there should be some trees you could climb. Kreedz found this aspect of the map to be particularly fun. He made more maps with a focus on this climbing aspect, and always named them with a kz_ tag as a sort of signature. Eventually they caught on and the kz_ tag stuck! It has become so popular that it has branched off into other games like **Half-Life** and even spawned its own game, [Kreedz Climbing](https://store.steampowered.com/app/626680/Kreedz_Climbing/). [Xtreme-Jumps](https://xtreme-jumps.eu/news.php) is currently the central hub for CS 1.6 Climb
 
 ## How Climb Works
 
-Climb works in the same way that bunny hopping does; pressing W or S combined with a strafe key causes the player to receive more velocity than simply holding W or S. Combining this with turning the player's camera in line with the strafe key, a player can get lots of momentum to jump to far platforms. This is normally called a **prestrafe**.
+### Strafing
+Strafing is the most common focus for maps and what sets Climb apart from the Bhop and surf game modes. Strafing can come in the form of long jumps and taking indirect lines. 
+Strafing works in the same way that bunny hopping does; pressing W or S combined with a strafe key causes the player to receive more velocity than simply holding W or S. Combining this with turning the player's camera in line with the strafe key, a player can get lots of momentum to jump to far platforms. This is normally called a **prestrafe**. This can be further improved by letting go of W in the air but continuing the mouse and strafe key pattern, called **airstrafing**. gaining speed before the jump and while in the air lets you travel futher. This is called a **long jump**. There are 2 other forms of longjump in 1.6 that are more advanced, [countjumps](https://kz-rush.com/en/page/countjump-physics) and [highjumps](https://kz-rush.com/page/highjump-physics). 
+
+Sometimes you may not be able to take a direct line due to an obstruction. The 2 most common forms of this is rounding a corner (often without being able to see the next platform) and 2 platforms separated by a thin wall. 
+
+In this case, air strafing is not used to gain speed but as a form of air control. The principle is the same but typically smooth turns at the cost of speed are needed (don't take out all your speed or you'll fall straight down!).
+
+### Bhop
+depending on the physics system used (1.6, CSGO Vanilla, KZT) and map design, bhopping can have reduced potential compared to the bhop game mode. In all physics systems it can be used as a way to preserve speed gains or minimize speed loss. 
```

Comment:
should specify the type of kz that's being talked about on this page. seems like most of this page is talking about 1.6 kz, but it's unclear. also "depending" should be capitalised ;)